src/dbnode/storage/stats/query_stats_default_tracker.go

```diff
@@ -53,27 +60,53 @@ func DefaultQueryStatsTracker(
 		MetricsScope().
 		SubScope("query-stats")
 	return &queryStatsTracker{
-		options:    queryStatsOpts,
-		recentDocs: scope.Gauge("recent-docs-per-block"),
-		totalDocs:  scope.Counter("total-docs-per-block"),
+		options:                   queryStatsOpts,
+		recentDocs:                scope.Gauge("recent-docs-matched"),
+		totalDocs:                 scope.Counter("total-docs-matched"),
```

Comment:
nit: `matched` isn't a great descriptor

Reply:
I wanted to avoid "read" since these docs aren't necessarily read yet, but rather matched in the index. You don't think matched conveys that?